Popup detail

BregornOriginal opened the pull request:
# New Feature to my portfolio: Popup added 😁

[![Img-popup.jpg](https://i.postimg.cc/y6gqCDr8/Img-popup.jpg)](https://postimg.cc/5jVKw2Vh)

## In this PR we work whit @KaskMIL on the feature "Pop-up"

### Now the website has the next features:
-  The button to check project details, showup a popup with details about the project
-  When the user clicks the close (X) button, the popup disappears
- The project cards now are added dynamically whit Javascript

> Technologies used in this feature
>
> HTML, CSS & JS

BregornOriginal committed:
Added icons and skeleton for popup-detail

BregornOriginal committed:
code ordered

BregornOriginal committed:
Added object details and first project card

BregornOriginal committed:
Updated the cards whit a for loop

KaskMIL committed:
changes

BregornOriginal committed:
projects from html deleted and work on popup

Added class names, id and buttons to popup section

BregornOriginal committed:
Update main.css whit styles for popup

BregornOriginal committed:
update position

BregornOriginal committed:
added hover for buttons

also added blur and disable scroll classes

BregornOriginal committed:
Update script

KaskMIL committed:
Merge branch 'popup-detail' of github.com:BregornOriginal/Julio-Gagliardi-Portfolio- into popup-detail

BregornOriginal committed:
added funcionality to buttons see project and popup

KaskMIL committed:
Add close btn popup functionality

BregornOriginal committed:
style to letter added

BregornOriginal committed:
Linters corrected

Ismailco commented on main.css:
```diff
@@ -320,6 +353,33 @@ ul {
+ 1050:1 Expected selector ".text-container ul li" to come before selector ".grid-section .secondary-card no-descending-specificity
+li" */
+
+.popup-section ul li {
```
- [ ] You may consider removing the space between the ```See Live``` and ```See source``` buttons so your design is as close as possible to the Figma design

![Screenshot_20220512_210131](https://user-images.githubusercontent.com/25227022/168158875-2fed82cc-0f33-40f5-8ff2-3d09c63a4ccd.png)

Ismailco commented on main.css:
```diff
@@ -19,6 +19,31 @@ ul {
+}
+
+/* disable-scroll for body for popup */
+.disable-scroll {
```
- [ ] On mobile the popup is not completed and can't scroll down, kindly fix this or remove the ```.disable-scroll``` so the user can reach the popup links

![Screenshot_20220512_210557](https://user-images.githubusercontent.com/25227022/168159363-2204ea15-01ed-44e2-8974-2c66b701f015.png)

Ismailco reviewed (changes requested):
### Hi @BregornOriginal,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Follow best practices
✔️ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] You have some typos in the README, please consider fixing them

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BregornOriginal committed:
Changes for review

- The space between the See live and See Source buttons now are more closer
- Now the scroll is avaible so the user can reach the popup links

m05e5 reviewed (approved):
# Hi @BregornOriginal , 

### Status: Approved ✔️
![goodjob](https://user-images.githubusercontent.com/60631456/130260474-3c2d5e5c-96df-43de-a0db-8d0059c10862.gif)


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
